lib/classes/API/V1/adapter.php

```diff
@@ -494,8 +494,7 @@ public function add_record(Application $app, Request $request)
                 throw new API_V1_exception_badrequest(sprintf('Invalid forceBehavior value `%s`', $request->get('forceBehavior')));
                 break;
         }
-
-        $app['border-manager']->process($session, $Package, $callback, $behavior);
+        $app['border-manager']->process($session, $Package, $callback, $behavior, $request->get('nosubdefs')==='');
```

Comment:
```php
$nosubdef = (($p = $request->get('nosubdefs')) === '' ? true : (bool) $p);
$app['border-manager']->process($session, $Package, $callback, $behavior, $nosubdef);
```
Just because if you set nosubdef to 1 (/url/to/api/embed?nosubdef=1)  it will build subdef :(